doc/rexml/tasks/mktocs.rb

```diff
@@ -0,0 +1,72 @@
+Dir.chdir(File.dirname(__FILE__)) do
```

Comment:
Could you define this as a `doc:toc:generate` Rake task in `Rakefile`?

Reply:
> 
> 
> Could you define this as a `rdoc:toc:generate` Rake task in `Rakefile`?

Sorry, I meant to say more (and better).  My initial question is Do we want this sort of thing?

If yes, everything else can come later:  correctness, robustness of TOC build, etc.

Also, Element is not completed, so there are some hanging issues there.

Reply:
It's acceptable.

It's better that RDoc provides the TOC feature as a build-in feature. For example, `[:toc:]` is expanded to TOC on rendering phase. But it seems that RDoc doesn't provide the TOC feature. (Could you request the feature to RDoc?)

So it's reasonable that we have TOC generator for us for now.

Reply:
> 
> 
> It's acceptable.
> 
This is a *lot* of work, so needs to be much more than 'acceptable.' Do you like the 'task' concept?

Reply:
Yes. I like the concept.

Reply:
Rakefile updated with task  tocs:generate.

Reply:
> 
> 
> It's acceptable.
> 
> It's better that RDoc provides the TOC feature as a build-in feature. For example, `[:toc:]` is expanded to TOC on rendering phase. But it seems that RDoc doesn't provide the TOC feature. (Could you request the feature to RDoc?)
> 
> So it's reasonable that we have TOC generator for us for now.

@kou, I have created the feature request: https://github.com/ruby/rdoc/issues/801

Reply:
Thanks!